### DISCUSSION TITLE
리뷰 공개 의견(?)

### DISCUSSION THREAD

**Discussion by hi-hyein** [ThinkAboutSoftware/AcademicConference#49] [General]
@jongfeel  @yoonucho  @Joseph-Cha 
먼저 죄송합니다 ㅠ_ㅠ
제 리뷰가 오늘 늦게 달릴 것 같은데 만약 두번째 리뷰가 완료되면 공개하시는건 어떨까요?

**Comment by Joseph-Cha** [comment]
혜인님보다 제가 더 늦을 수도 있습니다..ㅋㅋㅋㅋㅋㅋㅋㅋ

**Reply by Joseph-Cha** [reply]
방금전 멘토님의 리뷰를 남기는 것으로 제가 더 빨리 탈출하겠다는 자신감이 생겼습니당

**Reply by jongfeel** [reply]
혜인님 오늘 리뷰 진행하는 거면 혜인님이 마지막이니까 바로 리뷰 공개 해면 될 거 같은데요?

**Reply by hi-hyein** [reply]
ㅎㅎ혹시 댓글놀이?시간이 부족할까 걱정돼서 남겼는데
리뷰 작성 마무리하고 모두 공개했습니다~
모두 고생하셨습니다(_ _)
